### ISSUE TITLE
Audit all the status updates (especially on chain)

### ISSUE THREAD

**Issue by yaymukund** [boltlabs-inc/zeekoe#140]
In writing zkchannels-spec, we realized that Zeekoe may be missing some status updates to represent changes on chain. There's #76 and #77, which both have to do with on chain logic, but we should at some point step back and audit all the status updates.

**Comment by marsella** [comment]
I don't think this is necessary if we build testing infrastructure that verifies expected statuses after each operation. I honestly don't know if it's necessary anyway, we added all the necessary statuses ad-hoc as we integrated on-chain operations and verified at each step.
